CollAction/Models/AdminViewModels/ManageProjectViewModel.cs

```diff
@@ -69,7 +69,7 @@ public class ManageProjectViewModel
 
         [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. 'tag1;tag2'")]
         [MaxLength(30)]
-        [RegularExpression(@"^#[a-zA-Z_]\w*(;#[a-zA-Z_]+\w*)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
+        [RegularExpression(@"^[a-zA-Z_]+(;[a-zA-Z_]+)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
```

Comment:
If the intention is to disallow digits in the hashtags then the 'ErrorMessage' should be changed to:
"No spaces or digits are accepted, underscores are allowed. Separate multiple tags with a colon ';'."
N.B. I misspelled 'Separate' in the initial message, that should be corrected also.